src/dashboards/components/dashboard_index/DashboardsIndex.tsx

```diff
@@ -169,16 +156,19 @@ const mstp = (state: AppState) => {
     cloud: {limits},
   } = state
   const sortOptions = state.resources.dashboards['sortOptions']
+  const searchTerm = state.resources.dashboards['searchTerm']
```

Comment:
nit: We can probably consolidate these to a `getAll`, then just get the individual properties:

https://github.com/influxdata/ui/blob/master/src/resources/selectors/index.ts#L14

So something like:

```
const {sortOptions, searchTerm} = getAll<Dashboard>(
    state,
    ResourceType.Dashoards
  )
```